builds/posix/Makefile.in.plugins_examples

```diff
@@ -60,6 +60,16 @@ include     $(ROOT)/gen/make.shared.variables
 
 all: udrcpp_example dc_example kh_example crypt_app
 
+ifeq ($(PLATFORM),win32)
```

Comment:
If you need platform-depended modifications to makefiles - do them in prefix file, not here.